engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/FetchCachedResultsActor.scala

```diff
@@ -30,7 +30,7 @@ class FetchCachedResultsActor(cacheResultId: CallCachingEntryId, replyTo: ActorR
       case Success(Some(result)) =>
         val simpletons = result.callCachingSimpletonEntries map toSimpleton
         val jobDetritusFiles = result.callCachingDetritusEntries map { jobDetritusEntry =>
-          jobDetritusEntry.detritusKey -> jobDetritusEntry.detritusValue
+          jobDetritusEntry.detritusKey -> jobDetritusEntry.detritusValue.toString
```

Comment:
Not `.toString`, `.toRawString`. Hoping this is what's causing errors in this and the other PR.